tikv: add missing tikv configs (#20566)
pingcap/docs: tikv-configuration-file.md

+ Default value: `"60s"`
+ Minimum value: `"1s"`

### `end-point-memory-quota` <span class="version-mark">New in v8.2.0</span>

> * The maximum memory limit for TiKV Coprocessor requests. If this limit is exceeded, subsequent Coprocessor requests will be rejected with the error "server is busy."

Reviewer: ```suggestion
* The maximum amount of memory that TiKV Coprocessor requests can use. If this limit is exceeded, subsequent Coprocessor requests are rejected with the error "server is busy."
```

+ Limits the maximum CPU usage rate for TiKV periodic full compaction.
+ Default value: `0.1`, which means that the maximum CPU usage for periodic compaction processes is 10%.

### `follower-read-max-log-gap` <span class="version-mark">New in v7.4.0</span>

+ The maximum number of logs a follower is allowed to lag behind when processing read requests. If this limit is exceeded, the read request is rejected.
+ Default value: `100`

### `inspect-cpu-util-thd` <span class="version-mark">New in v7.6.0</span>

+ The threshold for determining whether a TiKV node is in a busy state during slow node detection.
+ Value range: `[0, 1]`
+ Default value: `0.4`, which means `40%`.

### `inspect-kvdb-interval` <span class="version-mark">New in v8.1.2</span>

+ The interval and timeout for checking the KV disk during slow node detection in TiKV. If KVDB and RaftDB share the same mount path, this value is overridden by `0` (no detection).
+ Default value: `2s`

### `min-pending-apply-region-count` <span class="version-mark">New in v8.0.0</span>

> + The maximum number of Regions busy applying Raft logs when TiKV starts. Raftstore only accepts leader transfers if the number of such Regions is below this value, reducing availability degradation during rolling restarts.

Reviewer: ```suggestion
+ The maximum number of Regions in the busy state of applying Raft logs when TiKV starts. Raftstore accepts leader transfers only when the number of such Regions is below this value, reducing availability degradation during rolling restarts.
```

+ Default value: `10`

### `request-voter-replicated-index-interval` <span class="version-mark">New in v6.6.0</span>

> + Controls the interval at which the Witness node periodically retrieves the replicated Raft log position from voting nodes.

Reviewer: ```suggestion
+ Controls the interval at which the Witness node periodically retrieves the replicated Raft log position from voter nodes.
```

> + Default value: 5 minutes

Author: ```suggestion
+ Default value: `5m`
```

### `slow-trend-unsensitive-cause` <span class="version-mark">New in v6.6.0</span>

> + When TiKV uses the SlowTrend detection algorithm, this parameter controls the sensitivity of latency detection. A higher value indicates lower sensitivity.

Reviewer: ```suggestion
+ When TiKV uses the SlowTrend detection algorithm, this configuration item controls the sensitivity of latency detection. A higher value indicates lower sensitivity.
```

+ Default value: `10`

### `slow-trend-unsensitive-result` <span class="version-mark">New in v6.6.0</span>

> + When TiKV uses the SlowTrend detection algorithm, this parameter controls the sensitivity of QPS-side detection. A higher value indicates lower sensitivity.

Reviewer: ```suggestion
+ When TiKV uses the SlowTrend detection algorithm, this configuration item controls the sensitivity of QPS detection. A higher value indicates lower sensitivity.
```

    + `true`: records information about WAL files in the MANIFEST file and verifies the integrity of WAL files during startup.
    + `false`: does not record information about WAL files in the MANIFEST file and does not verify the integrity of WAL files during startup.

### `enable-multi-batch-write` <span class="version-mark">New in v6.2.0</span>

> + Enables RocksDB write optimization, allowing the contents of WriteBatch to be written concurrently to the memtable, reducing write latency.

Reviewer: ```suggestion
+ Controls whether to enable RocksDB write optimization, allowing the contents of WriteBatch to be written concurrently to the memtable, reducing write latency.
```

+ Determines whether to generate empty log files for log recycling in Raft Engine. When it is enabled, Raft Engine will automatically fill a batch of empty log files for log recycling during initialization, making log recycling effective immediately after initialization.
+ Default value: `false`

### `compression-level` <span class="version-mark">New in v7.4.0</span>

> + Sets the compression efficiency of the LZ4 algorithm used by Raft Engine when writing Raft log files. Lower values indicate higher compression speed but lower compression ratio.

Reviewer: ```suggestion
+ Sets the compression efficiency of the LZ4 algorithm used by Raft Engine when writing Raft log files. A lower value indicates faster compression speed but lower compression ratio.
```
